doc/whats_new/v0.20.rst

```diff
@@ -80,11 +80,20 @@ Changelog
   where ``class_weight='balanced_subsample'`` failed with more than 32 classes.
   :issue:`12165` by `Joel Nothman`_.
 
+:mod:`sklearn.feature_extraction`
+...........................
+
+- |Fix| Fixed a regression in v0.20.0 where
+  :func:`feature_extraction.text.CountVectorizer` and other text vectorizers
+  could error during stop words validation with custom preprocessors
+  or tokenizers. :issue:`12393` by `Roman Yurchak`_.
+
 - |Fix| Fixed a bug affecting :class:`ensemble.BaggingClassifier`,
```

Comment:
this should be in ``sklearn.ensemble`` section

Reply:
It was actually in the correct section, but there was some conflict that apparently got resolved by Github but must have resulted in this weird rendering.

Rebased it to be sure, should be fine now.